compiler/noirc_evaluator/src/ssa/opt/loop_invariant.rs

```diff
@@ -176,6 +270,10 @@ impl<'f> LoopInvariantContext<'f> {
         // set the new current induction variable.
         self.current_induction_variables.clear();
         self.set_induction_var_bounds(loop_, true);
+        // The previous loop may have set that the current block is control dependent.
+        // If we fail to reset for the next loop, a block may be misadvertently labelled
```

Comment:
nit, but avoids cspell for `misadvertently`:

```suggestion
        // If we fail to reset for the next loop, a block may be inadvertently labelled
```